unable to link nodecloud-azure-plugin and nodecloud

## Expected Behavior
Getting an error saying package nodecloud-azure-plugin not found while linking nodecloud and nodecloud-azure-plugin

## Actual Behavior
```
$ npm link nodecloud-azure-plugin@1.0.0
npm ERR! code E404
npm ERR! 404 Not Found: nodecloud-azure-plugin@1.0.0

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\<user>\AppData\Roaming\npm-cache\_logs\2019-02-16T05_41_23_159Z-debug.log
```

## Steps to Reproduce the Problem

  1. git clone https://github.com/cloudlibz/nodecloud-azure-plugin
  2 cd nodecloud-azure-plugin
  4 npm link
  5 git clone https://github.com/cloudlibz/nodecloud
  6 cd nodecloud
  7 npm link nodecloud-azure-plugin

## Specifications

  - Version: node v8.13.0
  - Platform:  Windows 10
  - Subsystem:

--- a/package.json
+++ b/package.json
@@ -1,5 +1,5 @@
 {
-  "name": "nodecloud-azure",
+  "name": "nodecloud-azure-plugin",
   "version": "1.0.0",
   "description": "Azure Wrapper for NodeCloud Core",
   "main": "index.js",

